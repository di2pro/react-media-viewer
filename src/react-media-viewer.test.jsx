import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ReactMediaViewer from './index';

describe('React Media Viewer', () => {
  it('should match its snapshot', () => {
    const component = shallow(<ReactMediaViewer />);

    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
