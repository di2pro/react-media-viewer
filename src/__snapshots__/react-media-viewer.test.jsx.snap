// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Media Viewer should match its snapshot 1`] = `
<div>
  <h1>
    Hello World!!!
  </h1>
</div>
`;
